simple/util/dc_client.py

```diff
@@ -244,6 +244,50 @@ def resolve_entity_type(entity_dcids: list[str]) -> str:
   return common_entity_types.pop() if common_entity_types else ""
 
 
+def get_property_of_entities(entities: list[str],
+                             property_name: str) -> dict[str, str]:
+  return asyncio.run(get_property_of_entities_async(entities, property_name))
+
+
+async def get_property_of_entities_async(entities: list[str],
+                                         property_name: str) -> dict[str, str]:
+
+  chunks = chunked(entities, _BATCH_SIZE)
+
+  result: dict[str, str] = {}
+  async with AsyncClient(limits=_HTTPX_LIMITS, timeout=None) as client:
+    futures: dict[str, str] = [
+        _get_property_of_entities_chunk(client, chunk, property_name)
+        for chunk in chunks
+    ]
+    for result_chunk in await asyncio.gather(*futures):
+      result.update(result_chunk)
+
+  return result
+
+
+async def _get_property_of_entities_chunk(client: AsyncClient,
```

Comment:
maybe this could be in a separate PR?

Reply:
huh - this _was_ in a separate [PR](https://github.com/datacommonsorg/import/pull/309). Let me see how it got here.

Reply:
OK - rebased correctly now so this is gone for this PR.